frontend/javascripts/oxalis/shaders/main_data_fragment.glsl.ts

```diff
@@ -190,18 +191,21 @@ void main() {
   <% _.each(segmentationLayerNames, function(segmentationName, layerIndex) { %>
 
      // Color map (<= to fight rounding mistakes)
-     if ( length(<%= segmentationName%>_id) > 0.1 ) {
+     if ( length(<%= segmentationName%>_id_low) > 0.1 || length(<%= segmentationName%>_id_high) > 0.1 ) {
        // Increase cell opacity when cell is hovered
        float hoverAlphaIncrement =
          // Hover cell only if it's the active one, if the feature is enabled
```

Comment:
I think this has been removed before your PR, at least I don't see it

```suggestion
         // Hover cell only if it's the active one
```